torch/quantization/fx/quantize.py

```diff
@@ -26,7 +26,8 @@
     get_default_quant_patterns,
 )
 
-from .standalone_module import (
+from .observed_module import (
+    mark_observed_module,
```

Comment:
I think we can also define an `is_observed_module` and remove some of the validations in https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/quantize.py#L516-L522